catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -22,6 +30,11 @@
     }
   },
   "properties": {
+    "type": {
+      "description": "Pipeline type",
+      "$ref": "#/definitions/databaseMetadataConfigType",
+      "default": "DatabaseMetadata"
```

Comment:
Added the same trick we had for db connections to the SourceConfig classes. This way, when they get generated they already have the default property that allows pydantic to infer the source type we need. This smoothes out the class -> json -> class flow